usaspending_api/database_scripts/job_archive/management/commands/dev_4109_repair_awards.py

```diff
@@ -0,0 +1,231 @@
+"""
+Jira Ticket Number(s): DEV-4109
+
+Expected CLI:
+
+    $ python3 manage.py dev_4109_repair_awards
+
+Purpose:
+
+    Correct Faulty FKs and missing award records
+"""
```

Comment:
Is this script general use, or is there a time where we expect it to become obsolete?

Reply:
This script is to fix faulty data in the database. Optimally it would only need to be run once. However since I wasn't able to conclusively prove what the root cause is/was there is no guarantee the problem won't occur again. So the --inspect flag was added so the SQL can run on a daily basis to alert if the issue crops up again.

Between general use, and planned obsolescence it is most definitely the latter.

Reply:
I've seen "can be safely deleted after " comments on these scripts before. I don't know if we consider that general practice.

Reply:
Currently no.
Here is the mandatory header in the README: https://github.com/fedspendingtransparency/usaspending-api/blob/dev/usaspending_api/database_scripts/job_archive/README.md